test-clients/src/main/resource/contract/solidity/hip-206/safe/SafeHTS.sol

```diff
@@ -0,0 +1,175 @@
+            abi.encodeWithSelector(IHTS.dissociateTokens.selector,
+            account, tokens));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe multiple dissociations failed!");
+    }
+
+    function safeDissociateToken(IHTS token, address account) internal {
+        int32 responseCode;
+        (bool success, bytes memory result) = precompileAddress.call(
+            abi.encodeWithSelector(IHTS.dissociateToken.selector,
+            account, token));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe single dissociation failed!");
+    }
+
+    function safeTransferTokens(IHTS token, address[] memory accountIds, int64[] memory amounts) internal {
+        int32 responseCode;
+        (bool success, bytes memory result) = precompileAddress.call(
+            abi.encodeWithSelector(IHTS.transferTokens.selector,
+            token, accountIds, amounts));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe tokens transfer failed!");
+    }
+
+    function safeTransferNFTs(IHTS token, address[] memory sender, address[] memory receiver, int64[] memory serialNumber) internal {
+        int32 responseCode;
+        (bool success, bytes memory result) = precompileAddress.call(
+            abi.encodeWithSelector(IHTS.transferNFTs.selector,
+            token, sender, receiver, serialNumber));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe NFTs transfer failed!");
+    }
+
+    function safeTransferToken(IHTS token, address sender, address receiver, int64 amount) internal {
+        int32 responseCode;
+        (bool success, bytes memory result) = precompileAddress.call(
+            abi.encodeWithSelector(IHTS.transferToken.selector,
+            token, sender, receiver, amount));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe token transfer failed!");
+    }
+
+    function safeTransferNFT(IHTS token, address sender, address receiver, int64 serialNumber) internal {
+        int32 responseCode;
+        (bool success, bytes memory result) = precompileAddress.call(
+            abi.encodeWithSelector(IHTS.transferNFT.selector,
+            token, sender, receiver, serialNumber));
+        responseCode = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe NFT transfer failed!");
+    }
+
+    function safeCreateFungibleToken(IHTS.HederaToken memory token, uint initialTotalSupply,
```

Comment:
Can we add the [modifier](https://github.com/LimeChain/hedera-smart-contracts/blob/b1a7f33a4f49fbb2c0134f1420ffaec0d0a4ec06/hts-precompile/HederaTokenService.sol#L22) for default autoRenewPeriod to this library? This way we would be able to call the create functions with minimal required fields.